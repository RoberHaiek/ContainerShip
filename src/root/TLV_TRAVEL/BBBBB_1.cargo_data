#<uid>,<weight>,<dstPort>
container5,1,AAAAA
container6,1,DDDDD