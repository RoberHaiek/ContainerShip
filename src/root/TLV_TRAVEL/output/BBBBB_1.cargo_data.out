container4,unload,0,0,4
container3,unload,0,0,4
container2,unload,0,0,4
container4,load,0,0,4
container3,load,0,0,4
container5,load,0,0,3
container6,load,0,0,3