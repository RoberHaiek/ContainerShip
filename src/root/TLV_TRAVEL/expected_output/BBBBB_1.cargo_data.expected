container5,load,1,1,1
container6,load,1,1,1
container2,unload,1,1,1